ci/cases/yamls/gefs_replay_ci.yaml

```diff
@@ -0,0 +1,7 @@
+defaults:
+  !INC {{ HOMEgfs }}/parm/config/gefs/yaml/defaults.yaml
+base:
+  REPLAY_ICS: "YES"
```

Comment:
Some things to consider adding for this CI case:
```
FHMAX_GFS: 48
FHMAX_HF_GFS: 24
DO_EXTRACTVARS: "YES"         
FHOUT_HF_GFS: 3
```
If we are to add `FHOUT_HF_GFS` to `ci/cases/yamls/gefs_replay_ci.yaml`, a default `FHOUT_HF_GFS` will need to be added to `{{ HOMEgfs }}/parm/config/gefs/yaml/defaults.yaml`, and `FHOUT_HF_GFS=@FHOUT_HF_GFS@` will need to replace `FHOUT_HF_GFS=1` in `config.base`.

Reply:
Thanks, I'm currently testing.

Reply:
If our aim is to make this CI test like the reforecast, I also suggest setting `FHOUT_OCN_GFS` and `FHOUT_ICE_GFS` to 24 in `gefs_replay_ci.yaml`.  This would also mean adding default values for these variables in `{{ HOMEgfs }}/parm/config/gefs/yaml/defaults.yaml `and modifying `config.base` accordingly.

Reply:
@EricSinsky-NOAA Could you make a PR into my branch?

Reply:
Sure. Just created a PR into your branch. I can add these changes in my PR.